ci_framework/roles/tempest/tasks/main.yml

```diff
@@ -37,8 +37,11 @@
     cmd: "podman unshare chown 42480:42480 -R {{ cifmw_tempest_artifacts_basedir }}"
   when: not cifmw_tempest_dry_run | bool
 
-# > {{ cifmw_tempest_artifacts_basedir }}/tempest_output.log"
 - name: Run tempest
-  ansible.builtin.command:
-    cmd: "podman run -v {{ cifmw_tempest_artifacts_basedir }}/:/var/lib/tempest/external_files:Z {{ cifmw_tempest_image }}:{{ cifmw_tempest_image_tag }}"
+  containers.podman.podman_container:
+    name: tempest
+    image: "{{ cifmw_tempest_image }}:{{ cifmw_tempest_image_tag }}"
+    state: started
```

Comment:
Could You please try to wrapping in block/always, register 'tempest_container_outcome' and dump the stdout/stderr (if it still returns those) into cifmw-data/.* somewhere, as atm this output contains the most important info about tempest setup / execution and would be completely lost this way?